CHANGELOG

```diff
@@ -1,5 +1,9 @@
 devel
```

Comment:
CHANGELOG does not mention removal of existing optimizer rule "fulltext-index-optimizer".